package.json

```diff
@@ -167,7 +167,7 @@
         "husky": "^9.0.10",
         "latest-version": "^9.0.0",
         "lerna": "^8.1.2",
-        "lightningcss": "^1.24.0",
+        "lightningcss": "^1.19.0",
```

Comment:
If we want to pin the version, we should remove the `^`.

Reply:
its not needed the bug is above this minor version!

Reply:
But `^1.19.0` allows for versions up to `2.0.0`. Is this fine?